src/lib/slangroom/services.ts

```diff
@@ -22,48 +24,48 @@ export interface Service {
 	updated: string;
 }
 
-export const servicesContract = `
+export const getServices = async (): Promise<Response> => {
+	try {
+		const res = await slangroom.execute(
+			`
 Rule unknown ignore
 
 Given I have a 'string' named 'pb'
 When I write string 'api/collections/services/records?expand=issuer' in 'path'
 When I append 'path' to 'pb'
 Then print data
 Then I connect to 'pb' and do get and output into 'http_result'
-`;
-
-export const getServices = async (): Promise<Response> => {
-	try {
-		const res = await slangroom.execute(servicesContract, {
-			data: { pb: PB }
-		});
+`,
+			{
+				data: { pb }
+			}
+		);
 		return res.result.http_result.result;
 	} catch (e: any) {
 		console.log(e);
 		throw new Error(e);
 	}
 };
 
-export const serviceContract =(id:string)=> `
+export const getService = async (id: string): Promise<Response> => {
+	try {
+		const res = await slangroom.execute(
+			`
 Rule unknown ignore
 
 Given I have a 'string' named 'pb'
 When I write string 'api/collections/services/records/${id}?expand=templates' in 'path'
 When I append 'path' to 'pb'
 Then print data
 Then I connect to 'pb' and do get and output into 'http_result'
-`;
-
-export const getService = async (id:string): Promise<Response> => {
-	try {
-		const res = await slangroom.execute(serviceContract(id), {
-			data: { pb: PB }
-		});
+`,
+			{
+				data: { pb }
+			}
+		);
```

Comment:
Same as below, and I suggest to use meaningful variable names and not just the generic `http_result` but something like `services`